global.json

```diff
@@ -18,6 +18,7 @@
     }
   },
   "msbuild-sdks": {
+    "MSBuild.SDK.SystemWeb": "4.0.88",
```

Comment:
How will this get updated when new versions are available? Do we care at all?

Reply:
Not sure - probably manually until dependabot can handle global.json